src/utils/shortenText.js

```diff
@@ -0,0 +1,6 @@
+export default function shortenText(string) {
+  if (string.length > 25) {
+    return string.slice(0, 20) + "...";
+  }
+  return string;
+}
```

Comment:
If your text is only shown in one single line you could also consider this css option:
https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow?retiredLocale=de

The JS solution is fine, too.